eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -6424,4 +6425,50 @@ private function createContentWithReverseRelations(array $drafts)
 
         return $contentWithReverseRelations;
     }
+
+    public function testPublishVersionByUserWithLanguageLimitation()
+    {
+        $publishedContent = $this->createFolder(
+            [
+                self::ENG_US => 'Published US',
+                self::GER_DE => 'Published DE',
+            ],
+            $this->generateId('location', 2)
+        );
+
+        $draft = $this->contentService->createContentDraft($publishedContent->contentInfo);
+
+        $contentUpdateStruct = new ContentUpdateStruct([
+            'initialLanguageCode' => self::GER_DE,
+        ]);
```

Comment:
`ContentUpdateStruct` is not a part of SPI, so end-developer should not instantiate it directly.
What we should do here, to advocate good practice, is:
```php
$contentUpdateStruct = $contentService->newContentUpdateStruct();
$contentUpdateStruct->initialLanguageCode = self::GER_DE;
```